### PR TITLE
Changed the example host to magento2.dev

### PR DESCRIPTION
**Pull request by Zifius**

### CODE REVIEW

File: nginx.conf.sample [hunk 1]
```diff
@@ -11,7 +11,7 @@
 # }
 # server {
 #    listen 80;
-#    server_name mage.dev;
+#    server_name mage2.dev;
```
Comment (kandy) [inline nginx.conf.sample]: may be better to use full name: magento2.dev

Comment (Zifius) [inline nginx.conf.sample]: Agreed, updated

**Comment by daim2k5** [comment]
hey @Zifius please accept the Contributor License Agreement

**Comment by Zifius** [comment]
Hi Damian, done

**Comment by daim2k5** [comment]
@Zifius mhh doesn't work, can you try again?

**Comment by Zifius** [comment]
Ah sorry, thought you are talking about http://magento.com/legaldocuments/mca

**Comment by vancoz** [comment]
Hello @Zifius thank you for your contribution, but for now we going to reject it.
We can review our decision in case if you will provide good reason to accept this change.

Thank you.

**Pull request closed without merging**